packages/browser/src/core/user/index.ts

```diff
@@ -398,3 +398,53 @@ export class Group extends User {
     return undefined
   }
 }
```

Comment:
Is the idea that this utility class can also be shared with the User class -- either injected or through inheritance?

Reply:
Yes! either of those two. So either User inherits from `UniversalStorage` or we just inject the `_storage` instance to User class. Going to refactor those after we settle of overall approach.